backend/app/controllers/api/v1/courses_controller.rb

```diff
@@ -5,6 +5,11 @@ module V1
     class CoursesController < ApplicationController
       include Dry::Monads::Result::Mixin
 
+      def index
+        @courses = search_courses(Course.all)
```

Comment:
Let's limit it for something reasonable, like 100 max

Reply:
ready